### DISCUSSION TITLE
@this.set is not working in ie11

### DISCUSSION THREAD

**Discussion by pawan-fls** [livewire/livewire#3057] [7. Bugs (archived)]
### @this.set is not working in ie11

```
document.addEventListener('livewire:load', function () {
     @this.set("description","My Description");
})
```

**Context**
- Livewire version: [e.g. 2.2.3]
- Laravel version: [e.g. 8.32.1]
- Alpine version: [e.g. 2.7.3]
- Browser: [IE11]

**Comment by joshhanley** [comment]
@pawan-fls can you please provide some more information?

Is there any errors in the console?
Where in your component are you putting the above code?
Does it work when you run the same code in another browser?

**Comment by pawan-fls** [comment]
@joshhanley 

1.) Yes, there are errors in console:
Object doesn't support property or method 'set' on line 'window.livewire.find('kdchn9cdnde91yn3MxbK').set('description', "My Description")

2.) I am adding my code in component file: 
```
@push('scripts')
document.addEventListener('livewire:load', function () {
     @this.set("description","My Description");
})
@endpush
```

3.) Yes, it works fine in other browsers like chrome, mozilla.

**Comment by joshhanley** [comment]
@pawan-fls thanks for the extra details!

I believe the issue is IE11 doesn't support proxys.
Can you checkout this issue #2293? See if the workaround suggested there fixes the issue for you.

Hope this helps!
